commerce-cloud-code/app_yourshophere/cartridge/controllers/Privacy.js

```diff
@@ -0,0 +1,39 @@
+const server = require('*/server');
+
+server.post('CookieConsent', (req, res, next) => {
```

Comment:
could be cached right? or we leverage the uncached nature and send the customer back to the last click

Reply:
We could cache the request to show it, but as it is a post it is always the same. We could have another endpoint for display vs. action.